libminifi/include/core/FlowFile.h

```diff
@@ -224,6 +224,10 @@ class FlowFile {
 
   void setStoredToRepository(bool storedInRepository) {
     stored = storedInRepository;
+    if (!stored && nullptr != claim_)
+        {
+      claim_->decreaseFlowFileRecordOwnedCount();
```

Comment:
Initially I added this because I used storage in the repo as part of the count, but I removed this concept hence this isn't necessary. I will remove it, thanks.